kyverno/assets/logs/kyverno_tests.yaml

```diff
@@ -0,0 +1,76 @@
+id: "kyverno"
+tests:
+ -
+  sample: "2024-07-17T21:49:24Z\tERROR\twebhook-controller\twebhook/controller.go:269\tfailed to get lease\t{\"error\": \"lease.coordination.k8s.io \\\"kyverno-health\\\" not found\"}"
+  result:
+    custom:
+      caller: "webhook/controller.go:269"
+      error: "lease.coordination.k8s.io \"kyverno-health\" not found"
```

Comment:
We have a standard property for this kind of message: 
![image](https://github.com/user-attachments/assets/b7efce3d-e5da-49ab-96a3-76d0a0a8a74b)

Could you add a remapper for this one ? (You can also add the related facet)

Reply:
The domain in that doc refers to the groups in the facet correct? I saw source code for logger.name so I'm just assuming it is.